sdk/highlight-ruby/highlight/highlight.gemspec

```diff
@@ -1,30 +1,30 @@
 require_relative 'lib/highlight/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "highlight_io"
+  spec.name          = 'highlight_io'
   spec.version       = Highlight::VERSION
-  spec.authors       = ["Highlight"]
-  spec.email         = ["support@highlight.io"]
+  spec.authors       = ['Highlight']
+  spec.email         = ['support@highlight.io']
 
-  spec.summary       = "The Highlight SDK for Ruby"
-  spec.homepage      = "https://www.highlight.io"
-  spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
```

Comment:
I updated the min version here because this `rescue` is not valid in ruby 2.3
https://github.com/highlight/highlight/blob/et/lint-ruby/sdk/highlight-ruby/highlight/lib/highlight.rb#L45-L55

Can verify this by setting `TargetRubyVersion: 2.3` in `rubocop.yml` and running rubocop:

![Screenshot 2023-08-07 at 12 32 43 PM](https://github.com/highlight/highlight/assets/58678/66742f62-d464-4b65-9954-9d053108f291)

Reply:
2.3 is certainly EOL: https://www.ruby-lang.org/en/downloads/branches/

As is 2.6. We may want to consider setting the min version to be 3.0 but can save that for a follow up.